src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -18,6 +18,30 @@
   "additionalWizardParameters": {
     "$isMultiProjectTemplate$": "true"
   },
+  "supportedAuthentications": [
+    {
+      "auth": "None",
+      "authenticationType": "NoAuth",
+      "allowUnsecured": true
+    },
+    {
+      "auth": "Individual",
+      "authenticationType": "IndividualAuth",
+      "b2cAuthenticationOptions": "Local"
```

Comment:
Could you clarify how the tooling knows that `Individual` auth is only allowed if the "hosted" parameter is also true?

We discussed this before and I think you mentioned you expected that VS supports imposing this restriction, but I don't spot where it's defined in the config here. If we need to ask the project creation dialog people to add some feature for this, let's start that process ASAP.

Reply:
We might need to, I need to check with Phil